### PR TITLE
analytics for layers, print, measurement, bookmarks, external links, cyclomedia, and search

### PR DESCRIPTION
**Pull request by adieng01**
This PR add metrics to the layers control, print, measurement tools, bookmarking, search (bbl, address, and zoning districts) features.

### CODE REVIEW

**Comment by adieng01** [comment]
@hannahkates @trbmcginnis

**Comment by allthesignals** [comment]
@adieng01 Hi! Sorry I missed this. Let's figure out why the tests are failing & why your commits are signed this way. Also, let's squash the commits into one! Happy to help.

**Comment by allthesignals** [comment]
This looks great! Thank you so much @adieng01!

Let me know when you're ready to merge this.

File: app/components/is-in-viewport.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ export default class IsInViewportComponent extends Component.extend(InViewportMi
       eventCategory: 'Road View',
       eventAction: 'Entered Cyclomedia Viewport',
     });
-
+    console.log("entered view");
```
Comment (allthesignals) [inline app/components/is-in-viewport.js]: Let's nix this log! :)

**Pull request merged**